js/clinicalTimeline.js

```diff
@@ -51,6 +51,10 @@ window.clinicalTimeline = (function(){
       beginning = "0";
     }
 
+    var ticketValues = getTickValues(beginning, ending, getZoomLevel(beginning, ending, width * zoomFactor));
+    beginning = tickValues[0];
+    ending = tickValues[tickValues.length-1];
```

Comment:
The `var ticketValues` should prolly be `var tickValues`. Note also that the code still works, because tickValues was incorrectly defined by me as a global var. It should be changed in getTickValues() to `var tickValues` instead of tickValues. See also: http://www.w3schools.com/js/js_scope.asp